src/components/feedback/feedbackButton.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+const Button = styled.button`
+  position: fixed;
+  right: 0px;
+  bottom: 50%;
+  transform: translate(25%, 50%) rotate(-90deg);
+  background-color: #fff;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  color: #231c3d;
+  cursor: pointer;
+  font-size: 14px;
+  font-weight: 600;
+  padding: 6px 16px;
+  text-align: center;
+  text-decoration: none;
+  z-index: 9000;
+  &:hover {
+    background-color: #eee;
+  }
+  &:focus-visible {
+    outline: 1px solid #79628c;
+    background-color: #eee;
+  }
+`;
+
+export function FeedbackButton(props) {
+  return <Button {...props}>Feedback</Button>;
+}
```

Comment:
Can we type these props?

Reply:
this is prototype code, it's going to quickly go into the sdk and not even be built with styled at that point. low value change imo

Reply:
Alright 👍